components/brave_rewards/resources/ui/components/checkout/addFundsPanel/style.ts

```diff
@@ -0,0 +1,138 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const Subtitle = styled.div`
+  text-align: center;
+  font-family: ${p => p.theme.fontFamily.body}
+  font-size: 15px;
+  margin: -10px 0 25px 0;
+`
+
+export const CurrentBalance = styled.div`
+  padding: 15px 0 0;
+  border-top: solid 1px ${p => p.theme.color.separatorLine};
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+
+  > div:last-child {
+    text-align: right;
+    color: #009F61;
+    font-weight: 500;
+    font-size: 14px;
+  }
+
+  .balance-bat {
+    padding-left: 7px;
+    font-size: 16px;
+    font-weight: 600;
+  }
+
+  .balance-converted {
+    padding-left: 7px;
+    color: #9E9FAB;
+    font-size: 14px;
+  }
+`
+
+export const PurchaseButtonRow = styled.div`
+  margin: 14px 0 0;
+
+  button {
+    float: right;
+    width: 231px;
```

Comment:
we shouldn't set width as if long translation comes in it will go over. Outer should set button width or set min-width to this one